### PR TITLE
Add wgNaverSiteVerificationKey

### PR DESCRIPTION
**Pull request by paladox**

### CODE REVIEW

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -1719,6 +1719,15 @@
 		'help' => 'Site verification key for Pinterest Console.',
 		'requires' => [],
 	],
+	'wgNaverSiteVerificationKey' => [
+		'name' => 'Pinterest Site Verification Key (WikiSEO)',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php]: Pinterest?

File: ManageWikiSettings.php [hunk 2]
```diff
@@ -1719,6 +1719,15 @@
...
+		'from' => 'wikiseo',
+		'type' => 'text',
+		'overridedefault' => false,
+		'section' => 'seo',
+		'help' => 'Site verification key for Pinterest Console.',
```
Comment (Universal-Omega) [inline ManageWikiSettings.php]: Again

File: ManageWikiSettings.php (hunk 1 above)
Comment (paladox) [reply ManageWikiSettings.php]: that's an old commit.

Comment (Universal-Omega) [reply ManageWikiSettings.php]: Oh. Whoops. My bad.

**Pull request merged**